src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangasDto.kt

```diff
@@ -91,7 +91,7 @@ class ChapterDto(
     @SerialName("published_at") val date: String,
 ) {
     fun toSChapter(dateFormat: SimpleDateFormat) = SChapter.create().apply {
-        url = "/capitulo/$id"
+        url = "/capitulo/$id/"
```

Comment:
can it be done in the request? i think this will break the downloaded chapters

Reply:
nvm, downloads not dissapear

Reply:
Can't think of any others issues this could cause.

Cache misses? Would be relatively minor.